samples/browserslist/.browserslistrc

```diff
@@ -0,0 +1,30 @@
+# Comment
```

Comment:
The `samples/browserslist` directory also needs to match the capitalisation of the language entry. So with my previous comment in mind, this should be renamed to `samples/Browserslist`.

Reply:
<img width="536" alt="Screenshot 2020-12-01 at 02 57 18" src="https://user-images.githubusercontent.com/10416628/100679610-ed992800-3380-11eb-8fb5-63f98a3e7a04.png">

Hm.. Local test pass, but ci test fail )

Reply:
I assume you're using an OS with a case-insensitive filesystem (macOS or Windows). Our CI tests run on Linux, which uses a case-*sensitive* filesystem. This can cause confusion in version control.